src/Core/ClassInfo.php

```diff
@@ -400,7 +400,7 @@ public static function shortName($nameOrObject)
      */
     public static function hasMethod($object, $method)
```

Comment:
Could use a couple of unit tests for this behaviour

Reply:
Have added a range.